documents/scope.md

```diff
@@ -0,0 +1,20 @@
+# Scope and planning
+
+## Stage 1
+
+Estimated completion 2024.
+
+Based on discussion from [2024-04-02](https://docs.google.com/document/d/1a-777RetR4k2gZKHJ5pG3apWKt86jV7-vU-QChX7lv4/edit?usp=sharing) our stage 1 plan is:
+
+1. Focus on a small set of nouns, sourced from CLDR, e.g. month names, units etc to facilitate faster prototyping.
```

Comment:
I just realized something. There are a lot of languages. Some are more complicated than others. Perhaps start the focus on some languages with comprehensive coverage level in CLDR? I'm fine with starting the focus on English and Spanish, but perhaps some other languages can be considered depending on available resources/knowledge in the group.

Reply:
I added a new section and listed some potential candidates.